cognitedata/reveal
# fix: ensure models are loaded in the same order the add*-functions are called

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

## Description :pencil:

https://cognitedata.atlassian.net/browse/BND3D-3567

Fixes a "bug" where models are not added to the `.models` array in the order `.addModel` is called. 
This may not strictly be a bug, but it causes trouble e.g. when we use the model index in the React components for determining visibility.

## How has this been tested? :mag:



Linked into FDX and checked that models appear in the same order every time (and the right model is hidden every time).


## Test instructions :information_source:


- Link into FDX
- Open 3D page
- Hide one model using the layers-button
- Reload page
- Same model should be hidden still
- Reload again as many times it takes to be convinced


## Checklist :ballot_box_with_check:
